MdeModulePkg/Include/Library/DeviceStateLib.h

```diff
@@ -43,6 +43,14 @@ EFIAPI
 GetDeviceState (
   );
 
+/**
+Function to get InsecureDeviceState PCD

```

Comment:
I missed this before. Can we also not call out the PCD here? This should just describe the interface, not implementation details, so that if the library is overridden, it can change the implementation as it needs. And then this comment should be exactly the same as in the c file.

So something like

```c
/**
  Get the platform defined set of insecure device states. This can be used to measure insecure device
  states into the TPM or perform other required platform actions when the device enters an insecure state.

  @retval  The bitmask of insecure device states as defined by the platform.
**/
```